docs/index.html

```diff
@@ -8,113 +8,121 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Noto+Serif+JP&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="/css/index.css">
+  <link rel="stylesheet" href="/css/animation-book.css">
+    <link rel="stylesheet" href="https://matsumana07384.github.io/wedding-menu-table//css/animation-book.css">
```

Comment:
[must]
`/` が多いのとずれてるので削除お願いします（おそらく例文間違えた）

[imo]
他とスペース位置が合っていないので、合わせて頂きたく…

```suggestion
  <link rel="stylesheet" href="https://matsumana07384.github.io/wedding-menu-table/css/animation-book.css">
```

Reply:
あああ、ごめん。ありがとう！